scipy/interpolate/tests/test_interpolate.py

```diff
@@ -1248,36 +1250,36 @@ def binom_matrix(power):
             for a, b in np.random.uniform(-10, 20, (5, 2)):
                 int_a = pa.integrate(a, b)
                 int_d = pd.integrate(a, b)
-                assert_allclose(int_a, int_d, rtol=1e-13)
-                assert_allclose(pa_i(b) - pa_i(a), pd_i(b) - pd_i(a),
+                xp_assert_close(int_a, int_d, rtol=1e-13)
+                xp_assert_close(pa_i(b) - pa_i(a), pd_i(b) - pd_i(a),
                                 rtol=1e-13)
 
             roots_d = pd.roots()
             roots_a = pa.roots()
-            assert_allclose(roots_a, np.sort(roots_d), rtol=1e-12)
+            xp_assert_close(roots_a, np.sort(roots_d), rtol=1e-12)
 
     def test_multi_shape(self):
         c = np.random.rand(6, 2, 1, 2, 3)
         x = np.array([0, 0.5, 1])
         p = PPoly(c, x)
-        assert_equal(p.x.shape, x.shape)
-        assert_equal(p.c.shape, c.shape)
-        assert_equal(p(0.3).shape, c.shape[2:])
+        assert p.x.shape == x.shape
+        assert p.c.shape == c.shape
+        assert p(0.3).shape == c.shape[2:]
 
-        assert_equal(p(np.random.rand(5, 6)).shape, (5, 6) + c.shape[2:])
+        assert p(np.random.rand(5, 6)).shape, (5 == 6) + c.shape[2:]
```

Comment:
same thing again I may be missing?

```suggestion
        assert p(np.random.rand(5, 6)).shape == (5, 6) + c.shape[2:]
```